pkg/controllers/disruption/consolidation.go

```diff
@@ -42,6 +43,8 @@ const consolidationTTL = 15 * time.Second
 // consolidation is the base consolidation controller that provides common functionality used across the different
```

Comment:
I don't see that we added the note on the `do-not-consolidate` annotation here to remove it at v1